2nd-gen/pubsub-helloworld/functions/index.js

```diff
@@ -0,0 +1,77 @@
+/**
+ * Copyright 2022 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+"use strict";
+
+// [START import]
+const {onMessagePublished} = require("firebase-functions/v2/pubsub");
+const logger = require("firebase-functions/logger");
+// [END import]
+
+// [START helloWorld]
```

Comment:
I think we do want to prepend tag names with "v2."

As it is here, the snippet name is created as "js_helloWorld" which is rather general.

Conversely, a snippet name like "js_v2httpReadQueryParam" is more recognizable as a CF3 v2 snippet.

Reply:
Done